trifid/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+{{- $endpoint := .Values.trifid.sparql.endpoint -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "trifid.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "trifid.name" . }}
+    helm.sh/chart: {{ template "trifid.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "trifid.name" . }}
+      helm.sh/chart: {{ template "trifid.chart" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "trifid.name" . }}
+        helm.sh/chart: {{ template "trifid.chart" . }}
```

Comment:
The `"trifid.chart"` contains the version of the chart and would therefore trigger a redeployment on each upgrade (even if unnecessary). Please only add the `/name`, `/instance` and `/managed-by` labels